stdlib/public/core/Collection.swift

```diff
@@ -589,28 +589,6 @@ extension CollectionType
   }
 }
 
-extension SequenceType
-  where Self : _ArrayType, Self.Element == Self.Generator.Element {
-  // A fast implementation for when you are backed by a contiguous array.
-  public func _initializeTo(ptr: UnsafeMutablePointer<Generator.Element>)
-    -> UnsafeMutablePointer<Generator.Element> {
```

Comment:
This overload proved ineffective for e.g. `ArraySlice`, which has a nil `_baseAddressIfContiguous`. I instead added an `_initializeTo` implementation
in `Arrays.swift.gyb` that uses `_buffer._uninitializedCopy`.